packages/http-client/__tests__/basics.test.ts

```diff
@@ -39,7 +39,7 @@ describe('basics', () => {
   //     "url": "https://httpbin.org/get"
   //  }
 
-  it('does basic http get request', async done => {
+  it('does basic http get request', async () => {
```

Comment:
Got an error about passing `done` to an `async` callback.  The Jest version in this project is a couple major versions newer than what `http-client` had, so I figured that was a breaking change that went in.